VideoWeb/VideoWeb.AcceptanceTests/Features/JudgePrivateConsultation.feature

```diff
@@ -14,7 +14,6 @@ Scenario: Judge Can Enter The Consultation Room
 Scenario: Panel Member Can Enter The Consultation Room
```

Comment:
What is this test intending to do??  This is pre-existing test but the test scenario is all wrong.  Can you please look at the below video and then apply the fix accordingly? 
https://screenrec.com/share/hvPBjKZGaV